src/layout.js

```diff
@@ -21,7 +21,10 @@
 
 import {dev, user} from './log';
 import {isFiniteNumber} from './types';
+import {isExperimentOn} from './experiments';
 
+/** @private @const {string} */
+const UX_EXPERIMENT = 'amp-ad-loading-ux';
```

Comment:
What? This will get compiled into the main binary. Or you can do the opposite and define it here, then import that into the extension.